src/main/java/wiresegal/wob/coppermind/Wiki.kt

```diff
@@ -124,16 +191,16 @@ fun searchResults(searchInfo: String): Pair<Boolean, List<String>> {
     else {
         val search = searchInfo.split("\\s+".toRegex())
         val allArticles = wiki.search(searchInfo)
-                .map { it.first() }
+            .map { it.first() }
```

Comment:
Oof, compulsive auto code formatting strikes again, will undo these to match your style, of course :)
Have you heard of our lord and saviour, 4 space continuation indents, though?